home.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Flawless Shop</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" />
+</head>
+<body>
+  <header>
+    <nav class="nav">
+      <div class="nav-icons">
+        <ul>
+         <li><i class="fa fa-twitter"></i></li> 
+         <li><i class="fa fa-instagram"></i></li> 
+         <li><a href="">French</a></li>
+         <li><a href="">My page</a></li>
+         <li><a href="">Logout</a></li>
+        </ul>
+      </div>
+      <div class="nav-bar">
+        <a href="home.html"><img class="click" src="images/logo.png" alt="logo"></a>
+        <div class="nav-links">
+          <a class="LINK" href="about.html">About</a>
+          <a class="LINK" href="#products">Our Products</a>
+          <a class="LINK"href="#Artists">Artists</a>
+          <a href="" class="flawless">FLAWLESS</a>
+        </div>
+      </div>
+      <i class="fa fa-bars fa-2x" id="bars"></i>
+    </nav>
+    <div class="nav-menu">
+      <i class="fa fa-times fa-2x" id="closingIcon"></i>
+      <ul>
+        <li class="mobileLinks"><a href="home.html">Home</a></li>
+        <li class="mobileLinks"><a href="#products">Products</a></li>
+        <li class="mobileLinks"><a href="about.html">About</a></li>
+      </ul>
+    </div>
+    <div class="main">
+      <q>Hi there!</q>
+      <h1>With Flawless, Glow like never before!</h1>
```

Comment:
- [ ] Please consider adding the image inside the text to match the original design. This will make your application more professional and hence improved UI ☺️.

Your design 😎 
![image](https://github.com/mariegrace31/capstone-project/assets/99394374/770dfc3f-19dc-4e2a-b417-614b12c30b7f)

Original design 😎 
![image](https://github.com/mariegrace31/capstone-project/assets/99394374/150534a3-b35c-4272-9c90-1bd8fc3acc50)